tests/benchdnn/matmul/matmul.cpp

```diff
@@ -1082,6 +1083,8 @@ std::vector<data_kind_t> get_kinds_to_check(const prb_t *prb) {
     //       e.g. get_kinds_to_check_default_case
     std::vector<data_kind_t> check_kinds = {DST};
     if (!prb->attr.dropout.is_def()) check_kinds.push_back(DROPOUT_MASK);
+    if (prb->attr.scales.get(DNNL_ARG_DST).is_mx())
```

Comment:
I'd check for `is_def()` first just for extra safety in case a default changes or something.